cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Gov/DRepActivity.hs

```diff
@@ -7,6 +7,8 @@
 
 module Cardano.Testnet.Test.Gov.DRepActivity
   ( hprop_check_drep_activity
+  , makeActivityChangeProposal
+  , voteChangeProposal
```

Comment:
We shouldn't make dependencies between test modules IMO. These should go to a common module.

Reply:
Fixed here: https://github.com/IntersectMBO/cardano-node/commit/f1bf8b12e966ee93080f7ab2038b1c8a8e4eb191